lib/content-services/src/lib/content.module.ts

```diff
@@ -128,11 +130,18 @@ export class ContentModule {
             ngModule: ContentModule,
             providers: [
                 provideTranslations('adf-content-services', 'assets/adf-content-services'),
+                ContentAuthLoaderService,
                 {
                     provide: APP_INITIALIZER,
                     useFactory: versionCompatibilityFactory,
                     deps: [VersionCompatibilityService],
                     multi: true
+                },
+                {
+                    provide: APP_INITIALIZER,
```

Comment:
is it ok to have
`                    provide: APP_INITIALIZER,`
twice?
line 135 and 141

Reply:
yes, sure, there is this flag: multi: true and you can pass many providers for one token